docker/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '2'
 services:
   dtk_dev:
-    image: dalg24/dtk-stack:latest
+    image: dalg24/dtk-stack:17.07
```

Comment:
Goes in separate commit and will be dropped before merging.

Also I would vote for YY.MM.0